src/verify_cert.rs

```diff
@@ -151,14 +248,14 @@ fn check_signatures(
     loop {
         signed_data::verify_signed_data(supported_sig_algs, spki_value, &cert.signed_data)?;
 
-        if !crls.is_empty() {
+        if let Some(ref revocation_opts) = revocation {
```

Comment:
Style nit: I think the use of `ref` is not very idiomatic at this point. Instead, take a reference of the scrutinee (as in `&revocation`).

Reply:
Good call, thanks. Fixed up.